src/main/resources/assets/appliedenergistics2/lang/de_DE.lang

```diff
@@ -441,9 +494,67 @@ item.appliedenergistics2.ToolBiometricCard.name=Biometriekarte
 item.appliedenergistics2.ToolDebugCard.name=Dev.Debug-Karte
 item.appliedenergistics2.ToolReplicatorCard.name=Dev.Replikator-Karte
 
+// Commands
 commands.ae2.usage=Befehle verfügbar in Applied Energistics 2 - Verwende '/ae2 list' für eine Liste und '/ae2 help _____' für Hilfe zu einem Befehl.
 commands.ae2.permissions=Du hast nicht die erforderlichen Berechtigungen, um diesen Befehl auszuführen.
 commands.ae2.ChunkLogger=Schalted das Berichten von Chunk-Loading und Chunk-Unloading im Server-log um. ( OP )
-commands.ae2.ChunkLoggerOn=Chunk Logging ist nun aktiviert
-commands.ae2.ChunkLoggerOff=Chunk Logging is nun deaktiviert
+commands.ae2.ChunkLoggerOn=Chunk Logging ist aktiviert
+commands.ae2.ChunkLoggerOff=Chunk Logging ist deaktiviert
 commands.ae2.Supporters=Zeigt eine Liste aller AE2-Unterstützer
+
+// Achievements
+achievement.ae2.Compass=Meteoritenjäger
+achievement.ae2.Compass.desc=Fertige einen Meteoritenkompass
+achievement.ae2.Presses=Unbekannte Technologie
+achievement.ae2.Presses.desc=Finde eine Prozessorenpresse
+achievement.ae2.SpatialIO=Raumkoordinaten
+achievement.ae2.SpatialIO.desc=Fertige einen IO-Raumport
+achievement.ae2.SpatialIOExplorer=To boldly go
+achievement.ae2.SpatialIOExplorer.desc=Werde in einer Raumspeicherzelle gespeichert
+achievement.ae2.StorageCell=Besser als Truhen
+achievement.ae2.StorageCell.desc=Fertige eine Speicherzelle
+achievement.ae2.IOPort=Fertigungszellenshuffle
+achievement.ae2.IOPort.desc=Fertige einen ME-IO-Port
+achievement.ae2.CraftingTerminal=Eine (viel) größere Bank
+achievement.ae2.CraftingTerminal.desc=Fertige eine Fertigungskonsole
+achievement.ae2.PatternTerminal=Crafting Maestro
+achievement.ae2.PatternTerminal.desc=Fertige eine Schablonenkonsole
+achievement.ae2.ChargedQuartz=Schockierend
+achievement.ae2.ChargedQuartz.desc=Finde geladenen Certusquartz
+achievement.ae2.Fluix=Unnatürlich
+achievement.ae2.Fluix.desc=Stelle Fluixkristalle her
+achievement.ae2.Charger=Fluixproduktion
+achievement.ae2.Charger.desc=Fertige ein Ladegerät
+achievement.ae2.CrystalGrowthAccelerator=Beschleinuger ist eine Untertreibung
+achievement.ae2.CrystalGrowthAccelerator.desc=Fertige einen Kristallwachstumsbeschleuniger
+achievement.ae2.GlassCable=Fluix-Energie-Verbingung
+achievement.ae2.GlassCable.desc=Fertige ME-Glaskabel
+achievement.ae2.Networking1=Netzwerklehrling
+achievement.ae2.Networking1.desc=Erreiche 8 Kanäle mit Geräten in einem Netzwerk
+achievement.ae2.Controller=Netzwekschaltbrett
```

Comment:
"Netzwerksteuerung" could also work.

"Netzwerkknoten" might be a bit misleading as these are usually not limited to one per network, while controllers are limited to one per. So this should be reflected by the translation.